src/CreateScenarioButton/components/CreateScenarioDialog/CreateScenarioDialog.js

```diff
@@ -202,21 +203,24 @@ const CreateScenarioDialog = ({
         <Grid container spacing={2}>
           <Grid item xs={12}>
             <TextField
-             onChange={handleChangeScenarioName}
-             onBlur={handleChangeScenarioName}
-             autoFocus
-             id="scenarioName"
-             value={scenarioNameFieldValues.value}
-             error={scenarioNameFieldValues.hasError}
-             label={t(CREATE_SCENARIO_LABELS.DIALOG_SCENARIO_NAME)}
-             helperText={t(scenarioNameFieldValues.errorKey)}
-             fullWidth />
+              data-cy="scenario-name-textfield"
```

Comment:
Could be interesting to prefix the id by  `create-scenario-dialog` in order to narrow the possibilities to have several field with the same data-cy id?
Something like  `create-scenario-dialog-name-textfield`